mypy/checkexpr.py

```diff
@@ -1143,17 +1144,23 @@ def check_overload_call(self,
                                                  callable_name=callable_name,
                                                  object_type=object_type)
                 if not unioned_errors.is_errors():
-                    # Success! Stop early.
-                    return unioned_result
+                    # Success! But we need to see maybe normal procedure gives a narrower type.
```

Comment:
"need to see maybe normal procedure" -> "need to see if maybe the normal procedure"